src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -389,6 +389,37 @@ private static void close(final Closeable res) {
      * Stream monitor.
      */
     private static final class Monitor implements Callable<Void> {
+
+        /**
+         * Maximum number of log lines for a stack trace.
+         */
+        private static final int MAX_STACK_LENGTH = 1000;
+        /**
+         * Prefix "at ".
+         */
+        private static final String PREFIX_AT = "at ";
+        /**
+         * Prefix "Caused by".
+         */
+        private static final String PREFIX_CB = "Caused by";
+        /**
+         * Prefix "... ".
+         */
+        private static final String PREFIX_DOTS = "... ";
+        /**
+         * Format string for log statements.
```

Comment:
@mkordas we need those three `String` constants to identify if lines are part of a stack trace